dts/bindings/pinctrl/ite,it8xxx2-pinctrl-kscan-kbs.yaml

```
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE IT8XXX2 kscan pin controller kbs function node
+
+compatible: "ite,it8xxx2-pinctrl-kscan-kbs"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  pushpull-od-bit:
```

```suggestion
  pushpull-od-mask:
```
Mask better describes how this property is used.

---

Done